### PR TITLE
Df  dev  loggerfix

### PR DESCRIPTION
**Pull request by fils**
This is my logrus update.. a few unrelated items also slipped in there, sorry about that. I will try and get more disciplined.

As part of this I can to remove some of the logger functions and roll them back to log. Note, this is the first cut at this and we may wish to roll some things back to fmt.Print to get them into the console.

Also, at this time I only log to file, but I will try and add in a future option to go between stdout and file. It's just a simple change in the init function, so should be easy to add as an flag and or config option.

### CODE REVIEW

File: cmd/gleaner/main.go [hunk 1]
```diff
@@ -20,8 +22,25 @@ var viperVal, sourceVal, modeVal string
...
 
 func init() {
-	log.SetFlags(log.Lshortfile)
-	// log.SetOutput(ioutil.Discard) // turn off all logging
+	// Output to stdout instead of the default stderr. Can be any io.Writer, see below for File example
+
+	// name the file with the date and time
+	const layout = "2006-01-02-15-04-05"
+	t := time.Now()
+	lf := fmt.Sprintf("gleaner-%s.log", t.Format(layout))
```
Comment (nein09) [inline cmd/gleaner/main.go:30]: you could probably combine this line and the next into 
`Logfile := fmt.Sprintf("gleaner-%s.log", t.Format(layout))`

File: internal/millers/graph/graphng.go [hunk 2]
```diff
@@ -38,7 +37,8 @@ func GraphNG(mc *minio.Client, prefix string, v1 *viper.Viper) error {
 	// defer close(doneCh)           // Indicate to our routine to exit cleanly upon return.
 	isRecursive := true
 
-	// Spiffy progress line  (do I really want this?)
+	// Need count for Spiffy progress line  (do I really want this?)
+	// there has got to be a better way to get the count of objects in an object store
```
Comment (nein09) [inline internal/millers/graph/graphng.go:41]: I haven't seen one, unfortunately! But I don't know everything...

File: internal/organizations/org.go [hunk 3]
```diff
@@ -42,10 +42,10 @@ type Qset struct {
 // BuildGraph makes a graph from the Gleaner config file source
 // load this to a /sources bucket (change this to sources naming convention?)
 func BuildGraph(mc *minio.Client, v1 *viper.Viper) error {
-	var (
-		buf    bytes.Buffer
-		logger = log.New(&buf, "logger: ", log.Lshortfile)
-	)
+	//var (
```
Comment (nein09) [inline internal/organizations/org.go:45]: It looks like you might be able to delete these lines, and their subsequent friends - i guess `buf` just isn't ever used?

File: internal/summoner/acquire/headlessNG.go [hunk 4]
```diff
@@ -258,35 +187,21 @@ func PageRender(v1 *viper.Viper, mc *minio.Client, logger *log.Logger, timeout t
 	}
 	defer domContent.Close()
 
-	// Try LoadEventFired event... DOMContentEventFired  returns no data. client to buffer this event.
```
Comment (nein09) [inline internal/summoner/acquire/headlessNG.go:261]: ✨

File: internal/summoner/acquire/headlessNG.go [hunk 5]
```diff
@@ -316,7 +230,7 @@ func PageRender(v1 *viper.Viper, mc *minio.Client, logger *log.Logger, timeout t
 			});
 		}
 
-		function retry(fn, retriesLeft = 3, interval = 3000) {
+		function retry(fn, retriesLeft = 3, interval = 1000) {
```
Comment (nein09) [inline internal/summoner/acquire/headlessNG.go:233]: Did you mean to make that 1000? I think @valentinedwv had it at 3000, or am I misremembering?

**Review by nein09 — APPROVED** [review]
I just have a few small comments that are more stylistic than anything else, except for the one about the 1000 vs 3000 wait, which I may just be confused about. Nice!

**Review by valentinedwv — APPROVED** [review]
DV will need to update logging in the glcon after this is committed.
Is there a way to get some console, and file output at the same time?

**Pull request merged**